### PR TITLE
Removing warning in `visr()` when PARAMCD not present

### LINKED ISSUE #378: Remove 'PARAM' and 'PARAMCD' warning from `visr()`
The visualizations for time to event data in visR are amazing and will be used by many people, including those whose data does not follow CDISC conventions. We can do a couple of things to improve their experience, while not taking away from the experience of those working with CDISC data.

When data is used that does not have 'PARAM' and 'PARAMCD' columns, `visr()` prints a warning (see below) about the x-axis label. We don't need a warning to tell users what was not done. This should be handled similarly to other places in the package regarding labels. For example, if a column has a label, we use it. But if the column doesn't have a label, we use the variable name in `visr()`: we don't print a warning that the variable label wasn't used. It's already well documented in the the `visr()` help file, that if the 'PARAM' and 'PARAMCD' are present, their values will be used to construct the x-axis label. *Let's remove this warning*

```r
estimate_KM(data, strata = "sex") %>%
  visr()
#> Warning in visr.survfit(.): The x-axis label was not specified and could also
#> not be automatically determined due to absence of 'PARAM' and 'PARAMCD'.
```

### PR DESCRIPTION
**Pull request by ddsjoberg**
**What changes are proposed in this pull request?**
* When PARAM/PARAMCD columns are present in the source data, they are used to improve the x-axis label in `visr()`; this feature is clearly documented in the `visr()` help file. Previously, users received a printed warning if these columns were not present. This has been updated to no longer print a formal warning. (#378)

**If there is an GitHub issue associated with this pull request, please provide link.**
closes #378

--------------------------------------------------------------------------------

Checklist for PR reviewer

- [ ] PR branch has pulled the most recent updates from main branch. Ensure the pull request branch and your local version match and both have the latest updates from the main branch.
- [ ] If a new function was added, function should be included in `_pkgdown.yml`
- [ ] If a bug was fixed, a unit test was added for the bug check
- [ ] Run `pkgdown::build_site()`. Check the R console for errors, and review the rendered website.
- [ ] Code coverage is suitable for any new functions/features. Review coverage with `withr::with_envvar(new = c("NOT_CRAN" = "true"), covr::report())`. Before you run, begin a fresh R session without any packages loaded. 
- [ ] R CMD Check runs without errors, warnings, and notes
- [ ] `usethis::use_spell_check()` runs with no spelling errors in documentation
- [ ] Has `NEWS.md` been updated with the changes from this pull request under the heading indicating the latest version. If there is an issue associated with the pull request, reference it in parentheses at the end update (see `NEWS.md` for examples).
- [ ] Has the version number been incremented using `usethis::use_version(which = "dev")` 
- [ ] Approve Pull Request
- [ ] Merge the PR. Please use "Squash and merge".

### CODE REVIEW

**Comment by SHAESEN2** [comment]
We will need to discuss this within the team. The warning is part of our "educate over break" and good visualization practices.

**Comment by ddsjoberg** [comment]
> We will need to discuss this within the team. The warning is part of our "educate over break" and good visualization practices.

Sure, an group conversation is a good idea.

**Comment by ddsjoberg** [comment]
update documentation to mention PARAM xlabel

File: tests/testthat/test-visr_survfit.R [hunk 1]
```diff
@@ -18,7 +18,7 @@
 #' T2.8 No warning when `x_label` is `NULL` and the `survfit` object has a `PARAM` and a `PARAMCD` column.
 #' T2.9 No warning when `x_label` is `NULL` and the `survfit` object has a `PARAM` but no `PARAMCD` column.
 #' T2.10 No warning when `x_label` is `NULL` and the `survfit` object has no `PARAM` but a `PARAMCD` column.
-#' T2.11 A warning when `x_label` is `NULL` and the `survfit` object has no `PARAM` and no `PARAMCD` column.
```
Comment (SHAESEN2) [inline tests/testthat/test-visr_survfit.R:21]: can we add a test that 'Time' is added when paramcd and param are absent from the data?

Comment (ddsjoberg) [reply tests/testthat/test-visr_survfit.R:21]: This test already exists:

```r
testthat::test_that("T2.15 When `x_label` is `NULL` and the `survfit` object does not have a `PARAM` or `PARAMCD` column, the `x_label` is `NULL`.", {

  survfit_object <- adtte %>%
    dplyr::select(-c(PARAM, PARAMCD)) %>%
    visR::estimate_KM("SEX")

  suppressWarnings(gg <- survfit_object %>% visR::visr(x_label = NULL))

  testthat::expect_true("Time" %in% gg$labels$x)

})
```

**Pull request merged**